events/wmt21.md

```diff
@@ -212,29 +212,44 @@ Thursday, November 11, 2021
 
 ## Results
 
-### Shared Task: Machine Translation of News
-
-#### Translation into-English (SR+DC)
-*Systems ordered by DA score z-score; systems within a cluster are considered tied; lines indicate clusters according to Wilcoxon rank-sum test p < 0.05*
-
-|     |     |     |     |     |
-| --- | --- | --- | --- | --- |
-| **Language pair** | **Rank** | **Ave.** | **Ave. z** | **System** |
-| Czech→English | 1–2 | 77.8 | 0.111 | Facebook-AI |
-| German→English | 1–5 | 71.9 | 0.126 | Borderline |
-| Hausa→English | 1 | 74.4 | 0.248 | Facebook-AI |
-| Icelandic→English | 1 | 74.5 | 0.293 | Facebook-AI |
-| Japanese→English | 1 | 73.8 | 0.141 | HW-TSC |
-| Russian→English | 1–5 | 77.5 | 0.137 | NVIDIA-NeMo |
-| Chinese→English | 1–5 | 75.0 | 0.042 | NiuTrans |
-
-#### Translation out-of-English
-*Systems ordered by DA score z-score; systems within a cluster are considered tied; lines indicate clusters according to Wilcoxon rank-sum test p < 0.05*
-
-|     |     |     |     |     |
-| --- | --- | --- | --- | --- |
-
-
-
-
-[Summary of the results](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
+#### Full results of the shared tasks: [*Findings of the 2021 Conference on Machine Translation (WMT21)*](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
+
+### News Translation
+
+#### → English (segment rating with document context (SR+DC))
```

Comment:
Hi Lia!

Please notice that the Style Guide dictates '[do not use symbols as abbreviations](https://machinetranslate.org/style#acronyms-and-abbreviations)'. This is important for preparing the website to be machine translated in the future.